### ISSUE TITLE
Remove aggregation resource?

### ISSUE THREAD

**Issue by marten** [zooniverse/panoptes#2853]
AFAIK we're now running everything through Caesar when it comes to aggregations. Perhaps we can think about removing aggregations altogether from Panoptes. Clears a bit of memory from the DB for indexes if nothing else?

**Comment by camallen** [comment]
There is no need to for us to keep this data around now, though i'm wondering about 3rd parties that want a one API solution and they use this code base e.g. https://labelmars.net/

I'm leaning towards leaving the data model / table in place but clean up the data and rebuild the empty indexes. Thoughts?

**Comment by marten** [comment]
Technically we shouldn't remove the API calls, I guess. But if we can show that nobody is using them, I wouldn't be opposed to it just because it's not in line with API versioning. We could redirect it to Caesar's API. 

I've always felt that the argument for other parties possibly sometimes wanting to run their own Panoptes is a weak argument, and one that's constantly at odds with a highly scalable and performant system. Maybe we can provide a full docker-compose file that sets up and links various required services instead of ensuring that Panoptes provides a basic version of every feature?

**Comment by camallen** [comment]
> Maybe we can provide a full docker-compose file that sets up and links various required services instead of ensuring that Panoptes provides a basic version of every feature?

Sure - including a replica event stream to feed caesar?

**Comment by marten** [comment]
Yes, that'd be pretty simple, since ZooStream already has the concept of a configurable output. Currently [this](https://github.com/zooniverse/zoo_stream/blob/master/lib/zoo_stream/kinesis_publisher.rb) is the only one, but there could easily be one that delivers it as a number of synchronous HTTP calls to Caesar and ZooEventStats.

**Comment by camallen** [comment]
I've changed my mind (thanks LB) - i think we should remove this resource after some traffic analysis and ensuring nothing is using it anymore.

**Comment by camallen** [comment]
OK after checking logs it looks like looks like some apps still use this, specifically:
+ https://shaunanoordin.github.io/zooniverse-react-transcriber/app/index.html?env=production
+ https://preview.zooniverse.org/folgerdemo/

@shaunanoordin, does your hosted transcriber need the aggregations resource to stay alive? I don't know much about it but the fact it's hosted on your own github makes me think i can kill ignore it for this issue. 

I assume the folger demo is just old code that someone is still playing with, please correct me if wrong.

**Comment by shaunanoordin** [comment]
@camallen 
- The Zooniverse React Transcriber (for AnnoTate) basically stops being useful if `panoptes.zooniverse.org/aggregations` shuts down.
- That said, I'm not sure _if that app is still being used_ by the AnnoTate team. Last I checked, Chris was talking with Tate about what to do next, but that was 28 Sep 2018 - as far as I'm aware, we've already provided the data exports produced by that app.
- I can confirm that Shakespeare's World uses Caesar's aggregation end point to determined "completed lines"

**Comment by camallen** [comment]
Can we switch the transcriber app over to caesar if it's needed?

**Comment by shaunanoordin** [comment]
@camallen Can do, it should only take a day or two.
